docs/regional_coverage/region-description.md

```diff
@@ -1,8 +1,8 @@
 # Region description
 
-The considered regions of TIAM can be seen in figure 1. All are listed seperatly in table 1. 
+On a spatial level, TIAM considers 16 regions, in which some regions represent countries (e.g., the US) and others are aggregated (e.g., Africa); see Figure 1. Table 1 provides further information about the TIAM regions. 
 
-![TIAM Map](./figs/region-description.png) 
+![TIAM Map](./figs/region_description_2.png) 
```

Comment:
Please see my previous comment regarding these figures.

Reply:
Sorry need to delete the previous figure

Reply:
I see it perfectly alright when I open Visual Studio Code

Reply:
Why do we need "_2" in the file name? Just give the new file the same name as the old one and overwrite it. Or is there an intention to keep both files?

Reply:
Its just meant to be like a version control. So we don't loose old figures. However, if it is too confusing. Then I change it!

Reply:
Done!